packages/amplify-util-mock/src/__tests__/func/multi-function.test.ts

```diff
@@ -0,0 +1,49 @@
+import { prompter } from '@aws-amplify/amplify-prompts';
+import { start as lambdaServerStart } from '../../func';
+import { $TSAny } from 'amplify-cli-core';
+
+jest.mock('../../func');
+jest.mock('@aws-amplify/amplify-prompts');
+jest.mock('amplify-cli-core', () => ({
+  ...jest.requireActual('amplify-cli-core'),
+  FeatureFlags: {
+    getNumber: jest.fn(),
+  },
+}));
```

Comment:
looks like mocking core requires special handling , see https://github.com/aws-amplify/amplify-cli/blob/2e4cfb92b87499b1cdf2595e72b13b1857a60cb5/packages/amplify-util-mock/src/__e2e__/utils/index.ts#L20-L26 .

Reply:
thank you, i was able to resolve it